### DISCUSSION TITLE
handling database nb of passes for cutting

### DISCUSSION THREAD

**Discussion by skal** [arkypita/LaserGRBL#1482] [Q&A]
Hi,
pretty new witht his nice software, I did create a 20x20mm square in .svg file that I tried to cut with Ortur settings Plywood 3 passes.
I uncomment GCode multiples passes within the parameters section.
The generated path is the following:
M4 S0
S0 
G0X0.2Y0.2
S1000 
G1X20.2F100
Y20.2
X0.2
Y0.2
S0 
M5 S0
I don't know why, but it seems that is doesn't take into account the multiple path: I guess it's a user mistake ;-) Can someone give me some tip/link for that ?
I also don't understand why S0 is put after the M5 command (Spindle OFF), but this should another topic...
Thanks by advance,

     Samuel

**Comment by arkypita** [comment]
Actually the number of passes shown in "suggested settings" in material DB is only a hint that is not automatically applied.
You should manually set the number of passes into the box near job progress bar
